### PR TITLE
Meeting create

### PR DESCRIPTION
**Pull request by Moose3245**
Still shows "cant merge" after resolving conflicts. I pasted the fix in discord.

### CODE REVIEW

**Review by jc121F1 — APPROVED** [review]

File: .coveragerc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [run]
 branch = True
-omit = *__init__.py,*wsgi.py,*asgi.py,manage.py,*migrations*,*settings.py,*tests*
\ No newline at end of file
+omit = *__init__.py,*wsgi.py,*asgi.py,manage.py,*migrations*,*settings.py,*tests*,*/usr/lib/python3/dist-packages/*,*/venv/*
```
Comment (amir-rahim) [inline .coveragerc:3]: Why was this changed?

Comment (amir-rahim) [reply .coveragerc:3]: Should have no impact anyways, so it should be okay.

File: BookClub/templates/club_meetings.html [hunk 2]
```diff
@@ -9,100 +9,126 @@
...
+                                        <td>{{meeting.title}}</td>
+                                        <td>{{meeting.organiser.username}}</td>
+                                        <td>{{meeting.meeting_time}}</td>
+                                        <td>
+                                          <!--Insert URL for join meeting here-->
+
+                                              {% comment %} <form method="post" action="{% url 'join meeting url' club_url_name=club.club_url_name %}"> {% endcomment %}
+                                                {% comment %} {% csrf_token %} {% endcomment %}
+                                                <button type="submit" class="button is-info">Join Meeting</button>
+                                              {% comment %} </form> {% endcomment %}
```
Comment (amir-rahim) [inline BookClub/templates/club_meetings.html:105]: Remove Commented Out Form

File: .coveragerc (hunk 1 above)
Comment (amir-rahim) [reply .coveragerc]: Okay so looked at the action history for this PR, and saw the issues you were having was from Migrations Conflicts, this should now be sorted.
